src/core/services/managers/marker-manager.ts

```diff
@@ -61,6 +63,12 @@ export class MarkerManager {
     return this._markers.get(marker).then((m: Marker) => m.setZIndex(marker.zIndex));
   }
 
+  updateAnimation(marker: SebmGoogleMapMarker): Promise<void> {
+    return this._markers.get(marker).then((m: Marker) =>
+      m.setAnimation(google.maps.Animation[marker.animation])
```

Comment:
this can also be null/undefined:
```
if (typeof marker.animation === 'string) {
  m.setAnimation(google.maps.Animation[marker.animation]);
} else {
  m.setAnimation(marker.animation);
}
```